test/smoke_test.py

```diff
@@ -79,9 +80,20 @@ def main() -> None:
     print(f"torchvision: {torchvision.__version__}")
     print(f"torch.cuda.is_available: {torch.cuda.is_available()}")
 
-    # Turn 1.11.0aHASH into 1.11 (major.minor only)
-    version = ".".join(torchvision.__version__.split(".")[:2])
-    if version >= "0.16":
+    if re.match(r"\d+\.\d+\.\d+(?!a0)", torchvision.__version__):
+        try:
+            import torchvision.prototype as _
+        except ModuleNotFoundError:
+            pass
+        else:
+            raise AssertionError(
+                "torchvision.prototype available on a release version. "
+                "Run\n\n"
```

Comment:
Nit, the 2 new lines made it harder to see the suggestion for me

```suggestion
                "Run:\n"
```